orchestra/slack.py

```diff
@@ -50,9 +50,25 @@ def create_project_slack_group(project):
     return project.slack_group_id
 
 
+def _random_string():
+    return ''.join(
+        random.choice('abcdefghijklmnopqrstuvwxyz1234567890')
```

Comment:
I think it's preferred to do `import string` and `random.choice(string.ascii_lowercase + string.digits)`